toolchain/cc_toolchain_config.bzl

```diff
@@ -103,19 +103,20 @@ def _impl(ctx):
         ),
     ]
 
-    # System include path we need to setup for the downloaded clang
-    isystem_flags = feature(
-        name = "isystem_flags",
```

Comment:
Aren't `isystem_flags` necessary for `k8` ?

Reply:
No. This is a bit of a hack for now, but we don't have a sysroot for local clang so we just use /usr/include. We need to have a Linux x64 sysroot at some point, if we want to make the dev server properly platform independent.